library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss

```diff
@@ -16,13 +16,29 @@
 	}
 
 	#samigo-create-new-box, #samigo-create-import-box {
-		@extend .bs-callout-primary;
-		border-left-width: 1px;
-		margin-top: 20px;
+		margin-bottom: $standard-spacing;
+		padding: $standard-spacing;
+		border: 1px solid var(--sakai-border-color);
+		background: var(--sakai-background-color-2);
+
+		h2 {
+			margin: 0 0 15px 0; 	// to match the .form-group's spacing
```

Comment:
15 not 16?

Reply:
This is from Bootstrap, so it's their number "15".